common/views/components/MessageBar/MessageBar.js

```diff
@@ -0,0 +1,35 @@
+// @flow
```

Comment:
Wanna swap out the `BetaBar` with this `MessageBar` in Cardy? 🤡

Reply:
I was testing the Process.

Reply:
Done.